spec/system/candidate_interface/entering_details/degrees/candidate_entering_degrees_spec.rb

```diff
@@ -0,0 +1,203 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering a degree' do
+  include CandidateHelper
+
+  scenario 'Candidate enters their degree' do
+    given_i_am_signed_in
+    and_i_visit_the_site
+    when_i_click_on_degree
+    then_i_see_the_undergraduate_degree_form
```

Comment:
Suggestion: as above, collapse the above 3 into `when_i_view_the_degree_section`